### DISCUSSION TITLE
DLL Injetion does not work anymore after Update

### DISCUSSION THREAD

**Discussion by akrambak** [danielkrupinski/Osiris#4249] [Ideas]
Hi,

DLL Injetion does not work anymore after Update of yesterday.

![Capture d'écran 2024-04-03 015634](https://github.com/danielkrupinski/Osiris/assets/3879402/4d228118-e782-492e-a21a-b27e19864f31)

**Comment by Wojtek17779388** [comment]
almost every time the game updates the patterns change
